CMakeCache.txt

```diff
@@ -0,0 +1,329 @@
+# This is the CMakeCache file.
```

Comment:
You committed all these temporary crap from cmake. next time clean this shit.
just create a shell script like cleanup.sh where you remove all these temporary folders